src/app/components/event-card/event-card.scss

```diff
@@ -18,114 +18,106 @@
   }
```

Comment:
`event-card` doesn't relevant this PR too. Besides, as I can see here you do styles pretty with your changes. But it is another task for it. And it will be some conflicts. Why do you need changed `cards` in this PR?